cypress-io/cypress
# feat: re-enable Run All Specs

- Closes #24469, #24548

### User facing changelog
Brings back the "Run All Specs" feature via the opt in `experimentalRunAllSpecs` flag

### Additional details
Most of the wiring was still there so not a big lift. The UI is temporary and will be done in #24468 and #24467.

It feels much better to not kill the browser when clicking a new set to run. Could we ignore the memory implications since this feature is experimental?

### Steps to test
Check out this branch, run `yarn watch` in one terminal and `yarn cypress:open` in another once the watch is finished. Create a new Cypress project and add `experimentalRunAllSpecs: true` to the config. There will be an ugly button at the top of both spec-lists that will trigger the feature based on the current search filter.

### How has the user experience changed?


https://user-images.githubusercontent.com/25158820/201992983-e337f85b-50af-456e-ae19-a83c1a77b987.mov



### PR Tasks
